src/components/TwitchSidebar/TwitchSidebar.tsx

```diff
@@ -0,0 +1,55 @@
+import React, { FunctionComponent, useState, useEffect } from "react";
+
+import {
+  EuiPanel,
+  EuiImage,
+  EuiText,
+  EuiListGroup,
+  EuiListGroupItem,
+} from "@elastic/eui";
+import {  liveStreams } from './Streams'
+
+/* Styles */
+import "./_twitch-sidebar.scss";
+
+/* Types */
+import { TwitchStream } from "../../models/TwitchStream";
+
+export const TwitchSidebar: FunctionComponent = () => {
+  const [streams, setStreams] = useState<TwitchStream[]>([])
+  useEffect(() => {
+    liveStreams().then(data => {
+      setStreams(data)
+    })
+  }, [])
+  return (
+    <div className="matchup-content__right">
```

Comment:
and then you should be able to do `className={className}`

(im not 100% sure I got all that exactly right but you should be able to do something similar to this in order to pass className down)